### ISSUE TITLE
Qt and Gtk 2 theming is ugly

### ISSUE THREAD

**Issue by bmeznarsic** [Ferdi265/numix-solarized-gtk-theme#8]
OS: Solus 1.2.1 https://solus-project.com/
Desktop: Budgie
GTK version: GTK 3.20.9
Using theme version: NumixSolarized-20160919.tar.gz

I'm not sure if the gtk2 theming hasn't been implemented yet or if there's a bug (maybe me) but apps that use other toolkits seem to fall back on that really ugly default.
![screenshot from 2016-11-30 13-45-01](https://cloud.githubusercontent.com/assets/6306479/20752751/3a9cb76a-b703-11e6-9a27-b2d6fb673f6f.png)

**Comment by Ferdi265** [comment]
Strange. Because for me said applications work well and are themed accordingly.

I did remember having to muck around with Qt's config and a LibreOffice environment variable to get theming to work, but that was WAY before I created this theme.

![screenshot of VLC themed](https://i.imgur.com/8SrwwXj.png)

The settings I used (configured long before work on this theme started) on my ArchLinux box:

Said Qt config stuff:

In `qtconfig-qt4`, I changed it to use the GTK+ styling (fixes VLC and other Qt4 applications)
I also did some other stuff of which I don't yet have proof that they work (Qt5, don't use such applications)

Said LibreOffice env variable:

I had to set `SAL_USE_VCLPLUGIN` to `gtk` for LibreOffice 5.0 to correctly load the gtk style.
